var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -115,14 +131,23 @@ def install(self, spec, prefix):
                 'METIS_OPT=-I%s' % spec['metis'].prefix.include,
                 'METIS_LIB=%s' % metis_lib])
 
-        if '+mpi' in spec:
+        if 'mpi' in spec:
```

Comment:
Huh, never knew that. I wonder if we should do something to reduce the number of ways of specifying the same thing.